public/login.php

```diff
@@ -26,6 +26,7 @@
 
     if ($is_success == 0) {
         $_SESSION["alert_message"] = "Invalid email or Password!";
+        echo $_SESSION["alert_message"];
```

Comment:
Im guessing the the echo here wasn't intentional?

Reply:
yup removed